spec/game_runner_spec.rb

```diff
@@ -25,8 +25,10 @@
 
         expect(stdout.string).to include(grid_template)
       end
+    end
 
-      it "prompts a user for input, returns a message when given invalid input, and quits when input is valid" do
+    context "it prompts the user for input" do
```

Comment:
I think about `context` blocks as starting with `when`. For example `when a user ...`. The `it` at the beginning of the context implies it is just an example. Where you trying to make descriptions more efficient with blocks? Or is there some real behavioral context here?